### PR TITLE
React-Navigation & Redux

### PR DESCRIPTION
**Pull request by SethDonohue**
Added scaffolding for redux, base setup for react-navigation, routestack, routes, and stylesheets.

This PR ALSO adds the svgs we have for now, as well as the modules needed for the Redux and Navigation.

### CODE REVIEW

File: App.js
Comment (hRobert12) [inline App.js:25]: This system seems pretty simple, I like it!

File: src/routes/landing.js
Comment (hRobert12) [inline src/routes/landing.js:12]: This causes the "up" button to appear on the top corner. This may not be ideal if you have a nav bar at the bottom.

File: package.json
Comment (hRobert12) [inline package.json:25]: Took me a little while to figure out that you need to use node to install the react-navigation component to test the app. You may want to change the README to explain that.

**Review by hRobert12 — COMMENTED** [review]
Looking good. I left some comments for you to consider before you merge this.

File: src/routes/landing.js
Comment (SethDonohue) [reply src/routes/landing.js:12]: Yes, I haven't changed any of the layout compared to the examples I followed, so we can tweak that based on the wire-frames we get. I just wanted to get something up here for us to start with.

File: package.json
Comment (SethDonohue) [reply package.json:25]: Yeah, it's acting a bit weird, I am not sure why it doesn't put it in with npm install... It shows a warning for me when I test with 'react-native run-android'.
Did you need to install react-navigation separately or did you just run "npm install" to get the most recent package update? npm i should be run everytime you pull or update master.

Comment (SethDonohue) [reply package.json:25]: Ahhh... I used Yarn to install, that's probably why there was a warning, but if you run 'npm i' it should no longer show the warnings.
I'll still to using npm instead of yarn.

Comment (AndyCErnst) [reply package.json:25]: We should definitely use one or the other. I see this PR has a yarn.lock but the last one had an npm.lock. Yarn is considered better (more deterministic I think?) but either works.

Comment (SethDonohue) [reply package.json:25]: @AndyCErnst @hRobert12 @adamosimpson21 @djcroissant 
Which do you all work with? Yarn or NPM?

I noticed that when running the run-android for react native that it shows me warnings if i have not installed with NPM, but even though the items are there as i installed them with Yarn. I assume the emulator only checks for the package-lock.json and not the yarn-lock.json

This makes me think we should stick to NPM.

Comment (AndyCErnst) [reply package.json:25]: Sounds like NPM wins to me

**Review by AndyCErnst — CHANGES_REQUESTED** [review]
lgtm, but if we are going with NPM, please remove the yarn.lock file.

**Comment by SethDonohue** [comment]
OK, I removed the yarn lock and ran npm install so the package-lock is up to date.

**Review by AndyCErnst — APPROVED** [review]

**Pull request merged**
